Building Atom with Linter 1.11.4 pre-installed and it fails to activate

Hi,

I built Atom with this package pre-installed (by editing Atom's package.json, to include Linter 1.11.4 as a bundled package), now whenever I start Atom I get this error report:

**Atom Version**: 1.7.4
**System**: "Arch Linux"
**Thrown From**: [linter](https://github.com/atom-community/linter) package, v1.11.4


### Stack Trace

Failed to activate the linter package

```
At Cannot find module './linter.coffee'

Error: Cannot find module './linter.coffee'
    at Module._resolveFilename (module.js:338:15)
    at Function.Module._resolveFilename (/usr/share/atom/resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.module.exports.activate (/usr/share/atom/resources/app.asar/node_modules/linter/lib/main.js:154:20)
    at Package.module.exports.Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:183:20)
    at /usr/share/atom/resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/usr/share/atom/resources/app.asar/src/package.js:92:15)
    at /usr/share/atom/resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/usr/share/atom/resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/usr/share/atom/resources/app.asar/src/package-manager.js:538:21)
    at /usr/share/atom/resources/app.asar/src/package-manager.js:519:29
    at Config.module.exports.Config.transactAsync (/usr/share/atom/resources/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/usr/share/atom/resources/app.asar/src/package-manager.js:514:19)
    at PackageManager.module.exports.PackageManager.activate (/usr/share/atom/resources/app.asar/src/package-manager.js:497:46)
    at /usr/share/atom/resources/app.asar/src/atom-environment.js:754:28
```

### Commands

```
```

### Config

```json
{
  "core": {
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".flowconfig"
      ],
      "source.json": [
        ".arcconfig",
        "BUCK.autodeps"
      ],
      "source.lisp": [
        ".emacs",
        ".spacemacs"
      ],
      "source.python": [
        "BUCK"
      ],
      "text.html.basic": [
        "ejs",
        "htm",
        "html",
        "kit",
        "shtml",
        "tmpl",
        "tpl",
        "xhtml"
      ]
    },
    "disabledPackages": [
      "language-markdown",
      "atom-beautify",
      "atomic-chrome",
      "build",
      "Hydrogen",
      "git-plus",
      "language-asciidoc"
    ],
    "projectHome": "/home/fusion809/GitHub/mine",
    "themes": [
      "fusion-ui",
      "dark-bint-syntax"
    ]
  },
  "linter": {}
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

--- a/package.json
+++ b/package.json
@@ -2,6 +2,7 @@
   "name": "linter",
   "main": "./lib/index",
   "author": "steelbrain",
+  "readme": "./README.md",
   "version": "1.11.4",
   "description": "A Base Linter with Cow Powers",
   "repository": "https://github.com/steelbrain/linter",
@@ -47,7 +48,8 @@
     "eslint": "^2.4.0",
     "eslint-config-airbnb": "^6.0.2",
     "eslint-plugin-react": "^4.2.1",
-    "jasmine-fix": "^1.0.0"
+    "jasmine-fix": "^1.0.0",
+    "coffee-script": "^1.7.0"
   },
   "package-deps": [
     "linter-ui-default"

